### ISSUE TITLE
Benachrichtigungen per Email bei neuen Ereignissen

### ISSUE THREAD

**Issue by wnck** [florensh/wefactor#6]
Bei interessanten Ereignissen sollen interessierte Nutzer per Email benachrtichtigt werden, z.B.
- wenn ein Nutzer eine neue Variante für den eigenen Beitrag hochgeladen hat 
- wenn man sein Interesse für einen Beitrag bekundet hat und eine neue Version akzeptiert wurde,
-  wenn in einer seiner Gruppen ein neuer Beitrag eingestellt wurde

**Comment by florensh** [comment]
Wie können wir denn eMails versenden. Eigener Mailserver ist soweit ich weiß nicht erlaubt. Smtp-Server der Hochschule?

**Comment by wnck** [comment]
Richtig, Sie können den Mail-Server der Hochschule verwenden. Zum Testen können Sie ihren eigenen Account verwenden. Für den späteren Betrieb müssen wir einen separaten Account beantragen. Ich habe ein Programmbeispiel auf der WeFactor-Plattform in der Gruppe "HHN Developers" angelegt.

**Comment by florensh** [comment]
Feature ist entwickelt. Beantragen Sie einen Account?

**Comment by wnck** [comment]
Ja, mache ich.
